### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by utkarsh006**
- Made the README file concise for better readability of users.
- Formatted some lines of text and removed some of them.
- Added Contributors section and size of the repo.

## Screenshots

Screenshot (234)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,21 +19,27 @@ toolchain. Most developers will interact with Flutter via the [Flutter
 Framework](https://github.com/flutter/flutter), which provides a modern,
 reactive framework, and a rich set of platform, layout and foundation widgets.
 
-If you want to run/contribute to Flutter Web engine, more tooling can be
+To Run/contribute into Flutter Web engine, more tooling can be
```
Comment (fzyzcjy) [inline README.md:22]: uber-nit: Run -> run ;)

Comment (fzyzcjy) [reply README.md:22]: "To Run/contribute into Flutter Web engine" -> To run/contribute into Flutter Web engine?

Comment (fzyzcjy) [reply README.md:22]: anyway just uber nit

**Comment by utkarsh006** [comment]
@fzyzcjy Kindly Review Now.

**Comment by fzyzcjy** [comment]
Hi, I have no permission to *approve* the PR, and I cannot guess what Flutter team thinks about the PR :)

**Comment by utkarsh006** [comment]
> Hi, I have no permission to _approve_ the PR, and also impossible to guess what Flutter team thinks about the PR :🧡🧡)

Okay a ton thanks to review it ⭐

**Comment by utkarsh006** [comment]
@fzyzcjy Kindly refer this PR to someone who can merge it.

**Comment by utkarsh006** [comment]
@jason-simmons Kindly merge the PR.

**Review by chinmaygarde — CHANGES_REQUESTED** [review]
Thanks for the patch but I don't think this substantially improves the readme. We aren't sure all contributors want this displayed on the README and we don't know if the external service is going to be around.

**Comment by zanderso** [comment]
Thanks for the patch. After discussion, it seems that folks prefer the README.md as-is, so I'll go ahead and close this. Further discussion can go in the flutter-engine Discord. Cheers!

**Pull request closed without merging**